app/src/util/validation.ts

```diff
@@ -82,3 +82,13 @@ export const createSubCategory = z.object({
 });
 
 export type CreateSubCategory = z.infer<typeof createSubCategory>;
+
+export const createUserRequest = z.object({
```

Comment:
Could be moved to before the route definition to make it easier to see what data is available/ how the validation works (at least if it's only used in one place)